tools/tensorflow/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11
+RUN apt-get update
+RUN apt-get install -y build-essential
+RUN apt-get install -y libhdf5-dev
+RUN pip install h5py
+RUN pip install tensorflow-aarch64
```

Comment:
Does this actually work for x86 too?

Reply:
If it turns out that it doesn't, you can try just doing different behavior on each architecture by making use of the `TARGETARCH` build argument that Docker automatically provides: https://docs.docker.com/build/guide/multi-platform/#platform-build-arguments